lib/controllers/change-password.js

```diff
@@ -33,66 +33,80 @@ module.exports = function (req, res) {
   application.verifyPasswordResetToken(sptoken, function (err, result) {
```

Comment:
This needs to be inside the `helpers.handleAcceptRequest()`, else you'll handle the request even if `web.produces` is empty.